frontend/pages/journey/[id].vue

```diff
@@ -54,8 +54,8 @@ interface Journey {
 
 interface User {
     id: string;
-    firstName: string;
-    lastName: string;
+    display_name: string;
+    username: string;
     role: number;
```

Comment:
Wrong color for Darkmode 
![image](https://github.com/user-attachments/assets/9b42b8b2-6735-4602-b1fd-0017209b1422)